vyxal/structure.py

```diff
@@ -151,3 +152,34 @@ def __repr__(self):
         return (
             f"{type(self).__name__}({repr(self.branches)}, {self.modifier!r})"
         )
+
+
+def can_print(structures: list[Structure]) -> bool:
+    """Checks if the structures are printing instructions"""
+    for struct in structures:
+        if isinstance(
+            struct,
+            (
+                BreakStatement,
+                RecurseStatement,
+                FunctionCall,
+            ),
+        ):
+            continue  # These statements can't possibly print, so it's
+            # okay to just continue.
+        elif isinstance(struct, GenericStatement):
+            if struct.branches[0][0].value in list(",₴…") + ["¨,", "¨…"]:
```

Comment:
Hardcoding this... could cause problems.

Reply:
i agree

Reply:
well where else is it supposed to store the elements that print?

Reply:
You could extract it into a toplevel variable called `ELEMENTS_THAT_PRINT` or something so it's easier to hunt for.